plugins/generator-1.16.5/forge-1.16.5/tool.definition.yaml

```diff
@@ -7,9 +7,7 @@ templates:
     name: "@MODASSETSROOT/models/item/@registryname.json"
   - template: json/handheld_rod.json.ftl
     writer: json
-    condition:

```

Comment:
Revert this, before it was more readable

Reply:
But this is how item model files conditions are specified. I could change those too as well...

Reply:
You can change them in another PR, but keep this as it is as generally line-per-line is more readable

Reply:
Coming...